### PR TITLE
Test Steve

### PR DESCRIPTION
**Pull request by nameandlogo**
MySQL for twilio messages in public folder

### CODE REVIEW

File: vreasy/application/modules/vreasy/controllers/TwilioController.php [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+$sid = "AC5ef8732a3c49700934481addd5ce1659"; 
+$token = "{{ auth_token }}"; 
+$client = new Services_Twilio($sid, $token);
+ 
+$client->account->messages->sendMessage("+14158141829", "+15558675309", "Jenny please?! I love you <3", "http://www.example.com/hearts.png");
+*/
+
+
+
+use Vreasy\Models\Twilio;
```
Comment (brutuscat) [inline vreasy/application/modules/vreasy/controllers/TwilioController.php]: @nameandlogo this looks like it is missing in the PR

**Comment by brutuscat** [comment]
@nameandlogo thank you for submitting your test assignment. I reviewed your code and so far it looks fine, I like the fact that you adjusted yourself to our "code style".

The test has 2 other parts that I see missing in the PR (at least I haven't seen it!):
- Use Tests to confirm that twilio works and to cover agains regressions on the overall functionality.
- Process the twilio response and take an action (accept or reject the task)

I was wondering if you could comment how would you implement these with some code, maybe you forgot to push it?

**Comment by nameandlogo** [comment]
Hi Mauro
Just seen you comment 
I havn't actually set it up to run with twilio live for tesing that todo was more sudo code that would need to be added to use twilio for real, do you need it to run with a live account ?? 

I have added a table the sql is in the public folder that stores the messages and allows the assignee to reply (acccept reject) thats would be how it would work I think 

when the task is added it would call a function to send to twilio using the assignee phone number , then twilio would hit a url with the the responce referencing the task and the twilio phone number ? please let me know if that ok I am on skype maybe we can go over it some more ??

Thanks Steve

**Comment by brutuscat** [comment]
@nameandlogo 

> do you need it to run with a live account ??

No, that the whole point of testing! You do not run it against real live services. You "fake" it, eg: create "stub" requests. I see no test in your submission, you will see in the assignment doc that we ask for:

```
We expect that you use tests to simulate Twilio requests to a controller, and in general use tests to prove that the functionality is working.
```

We do this using the (Codeception)[http://codeception.com/] test framework. Checkout the readme, where we already show you how to run the (existing tests)[https://github.com/Vreasy/VrRecruit/tree/task-confirmation/tests/functional]

```
Check the test suite is working for you:

php vendor/codeception/codeception/codecept build
php vendor/codeception/codeception/codecept run --debug
It should output something like OK (2 tests, 4 assertions)

```

**Pull request closed without merging**